src/lib/pythongen.ml

```diff
@@ -559,8 +559,15 @@ let commandline_parse i (BoxedFunction m) =
               )
             | _ ->
               Line (
-                sprintf "parser.add_argument('%s', action='store', help='%s')"
-                  name (String.concat " " a.Idl.Param.description)
+                sprintf "parser.add_argument('%s', action='store', help='%s'%s)"
+                  name
+                  (String.concat " " a.Idl.Param.description)
+                  (match a.Idl.Param.typedef.ty with
+                   | Basic Int -> ", type=long" | Basic Int64 -> ", type=long"
+                   | Basic Int32 -> ", type=int"
+                   | Basic Bool -> ", type=lambda x: bool(distutils.util.strtobool(x))"
```

Comment:
I prefer `lambda x: json.loads(x.lower())` and avoid importing distutils

Reply:
Thanks, I've fixed and force-pushed